source/sourcing_and_processing_data/processors/extract_text.rst

```diff
@@ -0,0 +1,68 @@
+Extract Text processor
+======================
+
+This processor allows to extract any part of a text, a number or a combination of both into a new column. It's similar to the Replace via Regexp processor, except that instead of replacing the content in the same original column, a new column is created with the extracted text. The idea is indeed to put the part to extract in parenthesis. This part will then be extracted in a new column.
```

Comment:
I think that you should use `parentheses` instead of `parenthesis` for the plural form